examples/circom_full_flow.rs

```diff
@@ -57,9 +57,9 @@ fn main() {
 
     // initialize the Circom circuit
     let r1cs_path =
-        PathBuf::from("./folding-schemes/src/frontend/circom/test_folder/external_inputs.r1cs");
+        PathBuf::from("./folding-schemes/src/frontend/circom/test_folder/full_flow_example.r1cs");
     let wasm_path = PathBuf::from(
-        "./folding-schemes/src/frontend/circom/test_folder/external_inputs_js/external_inputs.wasm",
+        "./folding-schemes/src/frontend/circom/test_folder/full_flow_example_js/full_flow_example.wasm",
```

Comment:
I was thinking if this example could not reuse the already existing `folding-schemes/src/frontend/circom/test_folder/with_external_inputs.circom` circuit instead of duplicating it into `full_flow_example.circom`, since the circuits are identical exept for the 'IsZero' template usage.
With the idea to do it similarly to what was done before this PR, where the `circom_full_flow.rs` example reused the circom test circuits.

But when I've tried the `example/circom_full_flow.rs` example using the `with_external_inputs.circom` circuit it fails with `SynthesisError(AssignmentMissing)`.

To reproduce it just change these couple of lines to load the `with_external_inputs.{r1cs/wasm}` instead of `full_flow_example.{r1cs/wasm}` and run the `examples/circom_full_flow.rs` example.

Reply:
Weird, I don't get `Err(Assignment)` missing but `Err(Unsatisfiable)` which is expected - this is related to that [issue](https://github.com/privacy-scaling-explorations/sonobe/issues/104) and is why I switched to a circuit that can be satisfied with a zero vector assigned.

Reply:
You're right! My bad, I was using somehow a wrong version of the branch!

Reply:
Now that the sanity-check does not trigger the error (since the commit https://github.com/privacy-scaling-explorations/sonobe/pull/101/commits/c278b931e96af480292407d38626ff40d55d9aa9 ), how would you see to update this to reuse the `with_external_inputs.circom` related files and removing the (almost) duplicate `full_flow_example.circom`?

Reply:
Yup, good suggestion. Done here [bdf6578](https://github.com/privacy-scaling-explorations/sonobe/pull/101/commits/bdf657801ee6415c78e7a320f3a9588cedb36367)